kubelet/datadog_checks/kubelet/cadvisor.py

```diff
@@ -0,0 +1,169 @@
+# (C) Datadog, Inc. 2010-2018
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+"""kubernetes check
+Collects metrics from cAdvisor instance
+"""
+# stdlib
+from fnmatch import fnmatch
+import numbers
+from urlparse import urlparse
+
+# 3p
+import requests
+
+# check
+from .common import FACTORS, tags_for_docker, tags_for_pod, is_static_pending_pod, get_pod_by_uid
+
+NAMESPACE = "kubernetes"
+DEFAULT_MAX_DEPTH = 10
+DEFAULT_ENABLED_RATES = [
+    'diskio.io_service_bytes.stats.total',
+    'network.??_bytes',
+    'cpu.*.total']
+DEFAULT_ENABLED_GAUGES = [
+    'memory.usage',
+    'filesystem.usage']
+DEFAULT_POD_LEVEL_METRICS = [
+    'network.*']
+
+NET_ERRORS = ['rx_errors', 'tx_errors', 'rx_dropped', 'tx_dropped']
+
+LEGACY_CADVISOR_METRICS_PATH = '/api/v1.3/subcontainers/'
+
+
+class CadvisorScraper():
```

Comment:
Why not using the *new style* Python 2.7 class:
```python
class CadvisorScraper(object):
```
[WDYT ?](https://stackoverflow.com/questions/4015417/python-class-inherits-object?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa)